src/Lithnet.GoogleApps/GmailServiceRequestFactory.cs

```diff
@@ -134,14 +175,14 @@ public void RemoveSendAs(string id)
         {
             id.ThrowIfNotEmailAddress();
 
-            GmailService service = this.GetService(id);
-            ListSendAsResponse result = service.Users.Settings.SendAs.List(id).ExecuteWithRetry(RetryEvents.BackoffOAuthNotFound, this.RetryCount);
+            GmailService service = this.GetService(id, DefaultTimeout);
+            ListSendAsResponse result = service.Users.Settings.SendAs.List(id).ExecuteWithRetry(RetryEvents.BackoffOAuth, this.RetryCount);
 
             if (result.SendAs != null)
             {
                 foreach (SendAs item in result.SendAs.Where(t => !t.IsPrimary ?? false))
                 {
-                    service.Users.Settings.SendAs.Delete(id, item.SendAsEmail).ExecuteWithRetry(RetryEvents.BackoffOAuthNotFound, this.RetryCount, 5);
+                    service.Users.Settings.SendAs.Delete(id, item.SendAsEmail).ExecuteWithRetry(RetryEvents.BackoffOAuth, this.RetryCount, 5);
```

Comment:
Any reason to remove the 'NotFound' policy?

Reply:
I think the instinct was to not retry the Delete request if the SendAs address does not exists (then we cannot delete it anyway). I may be missing something here though. Should I revert?